webapp/javascript/components/TimelineChart/Selection.plugin.ts

```diff
@@ -1,16 +1,35 @@
-/* eslint-disable */
+/* eslint-disable eqeqeq */
```

Comment:
typo?

Reply:
no, it's https://eslint.org/docs/latest/rules/eqeqeq

I didn't take a risk to transform all `==` to `===`

Reply:
ohh

can you add a message explaining that, bc i can see most people ignoring it haha
https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-comments

Reply:
updated

Reply:
Not sure I understood, i was asking for a comment explaining the use of `eqeqeq`. As I linked above, you use `--` to delimit the comments. Eg
```
/* eslint eqeqeq: "off", curly: "error" -- Here's a description about why this configuration is necessary. */
```

Reply:
ah, really, please check it one more time 😅